src/Core/Auth/Services/Implementations/SsoConfigService.cs

```diff
@@ -78,8 +78,13 @@ public async Task SaveAsync(SsoConfig config, Organization organization)
 
             resetPolicy.Enabled = true;
             resetPolicy.SetDataModel(new ResetPasswordDataModel { AutoEnrollEnabled = true });
-
             await _policyService.SaveAsync(resetPolicy, _userService, _organizationService, null);
+
+            var ssoRequiredPolicy = await _policyRepository.GetByOrganizationIdTypeAsync(config.OrganizationId, PolicyType.RequireSso) ??
+                              new Policy { OrganizationId = config.OrganizationId, Type = PolicyType.RequireSso, };
+
+            ssoRequiredPolicy.Enabled = true;
+
```

Comment:
issue: We need to save the `ssoRequiredPolicy` with the `_policyService`. Something like:

```c#
await _policyService.SaveAsync(ssoRequiredPolicy, _userService, _organizationService, null);
```

Reply:
Fixed! I accidentally deleted that with an errant keystroke before committing, after testing locally. Thank you for catching.